android-components/plugins/dependencies/src/main/java/ApplicationServices.kt

```diff
@@ -0,0 +1,38 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+// These lines are generated by android-components/automation/application-services-nightly-bump.py
+val VERSION = "114.20230420185101"
+val CHANNEL = ApplicationServicesChannel.NIGHTLY_STAGING
+
+object ApplicationServicesConfig {
+    val version = VERSION
+    val channel = CHANNEL
+    val groupId = when (channel) {
+        ApplicationServicesChannel.RELEASE -> "org.mozilla.appservices"
+        ApplicationServicesChannel.STAGING -> "org.mozilla.appservices"
+        // Use a different groupid for the nightly channel.  This is a bit of a
+        // workaround for the fact that we can't yet upload to the nightly maven
+        // repos yet.  Once we can do that, maybe we should use the same groupid in
+        // all cases.
```

Comment:
nit: We could remove the comment about the workaround, because it's fine to have a separate groupId. That's what we do in GV as well. :)